src/pages/Account/Account.tsx

```diff
@@ -2,12 +2,59 @@ import Layout from "../../components/Layout/Layout";
 import "./Account.scss";
 import Header from "../../components/Header/Header";
 import Button from "../../components/Button/Button";
+import { auth } from "../../firebase";
+import { User, signOut } from "firebase/auth";
+import { useNavigate } from "react-router-dom";
+import { getUser } from "../../utils/firebaseSnapshots";
+import { useEffect, useState } from "react";
 import placeHolderPFP from "/src/assets/images/placeHolderPFP.svg";
 import InputBox from "../../components/InputBox/InputBox";
 
-const Account = () => {
-  const handleSignOut = () => {
-    return;
+type AccountProps = {
+  setUser: (user: User | null) => void;
+};
+
+const Account = ({ setUser }: AccountProps) => {
+  const [firstName, setFirstName] = useState<string>("");
+  const [lastName, setLastName] = useState<string>("");
+  const [email, setEmail] = useState<string>("");
+
+  const navigate = useNavigate();
+  const userId = auth.currentUser?.uid as string;
```

Comment:
I think this is not needed, you can pass the user state from App into this page then you will have access to the UUID and you won't have to do the ? as it will only be able to load that page if a user is true.